### ISSUE TITLE
Tyrant can get stuck in pit

### ISSUE THREAD

**Issue by necessarily-equal** [interstellar-oasis/map-nexus6_src.dpkdir#2]
Example:

![Screenshot from 2021-11-23 12-58-02](https://user-images.githubusercontent.com/59283660/143020296-70f43cb8-94ec-4643-b3c4-26839c1d7aec.png)

Even players can't get out in this situation. I wonder what would be the correct fix: adding an elevator platform?

**Comment by ghost** [comment]
This also happens on yocto, and in both cases it's hard to get down as a tyrant. I think it's more likely the bot evolved in here.

**Comment by necessarily-equal** [comment]
In this specific case, I did jump in the pit to get the tyrant to follow me. Falling in the pit as a tyrant is difficult indeed, but still possible. I'd say we should always allow a way to do exit the pit, even if it can be a low-priority bug, because people are not supposed to know about /kill.

Le 23 novembre 2021 16:32:03 GMT+01:00, "Morel Bérenger" ***@***.***> a écrit :

**Comment by ghost** [comment]
Agree, but then you should place that same bug in all maps. Considering tyrants have no special moves, the only way would be to place an elevator in maps featuring such "bot traps", but even this would not fix the AI issue since AIs don't know how to use elevators (implementing it would be very nice, but this will require quite the work, especially for elevators which are triggered by a specific button, like in spacetrack, some parpax's, the one at "elevator room" in karith station...)
